extensions/amp-analytics/0.1/events.js

```diff
@@ -406,3 +478,10 @@ export class VisibilityTracker extends EventTracker {
     listener(new AnalyticsEvent(target, eventType, state));
   }
 }
+
+/** @const @private */
+const SUPPORT_WAITFOR_TRACKERS = {
```

Comment:
So, we don't need the enum above then, right? Otherwise this seems repetitive...